test/e2e/tags.py

```diff
@@ -0,0 +1,25 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Utilities for working with tags"""
+
+from typing import List
+
+ACK_SYSTEM_TAG_PREFIX = "services.k8s.aws/"
+
+def clean(tags: List[object]) -> List[object]:
+    """Returns supplied tags collection, stripped of ACK system tags.
+    """
+    return [
+        t for t in tags if not t['key'].startswith(ACK_SYSTEM_TAG_PREFIX)
```

Comment:
Instead of ignoring the ACK_SYSTEM_TAGS, we should assert the tag Keys are present. Atleast the keys since value can keep changing

Reply:
yeah, I wish to do it for all the controllers together from a common test-infra lib.
Currently, I was just fixing the tag assertion failures.